### ISSUE TITLE
Automatic Card creation on login do not set user_created and user_modified fields

### ISSUE THREAD

**Issue by batopa** [bedita/bedita#619]
After #573 when a user without Card associated logs in to backend, a new Card is correctly created but user_created and user_modified are not populated.

Those fields should be set to system user #618. I propose to add to form fields on login a field *data[_useSystemUser]* and in `User::afterSave()` if the user hasn't a Card associated with it and that field is populated then set *user_created* and *user_modifed* to system user.

@stefanorosanelli @fquffio other ideas?

**Comment by stefanorosanelli** [comment]
Ok for me

**Comment by fquffio** [comment]
We could find out if its a login attempt by checking against presence of key `login` in `$this->data` array. I'd rather not add a hidden field to a login form, as this would be too easily exploitable. @batopa @stefanorosanelli ?

**Comment by batopa** [comment]
@fquffio mmm, maybe it was already solved with #618. Isn't it?

**Comment by fquffio** [comment]
@batopa Did some testing, and it looks like you are right once again.

**Comment by batopa** [comment]
ok, closing
